lib/broccoli/packager.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const concat = require('broccoli-concat');
+const mergeTrees = require('broccoli-merge-trees');
+const DEFAULT_CONFIG = require('./default-config');
+
+/*
+  Creates an object with lists of files to be concatenated into `vendor.js` file.
+  Given a map that looks like:
+  {
+    'assets/vendor.js': [
+      'vendor/ember-cli-shims/app-shims.js',
+      'vendor/loader/loader.js',
+      'vendor/ember-resolver/legacy-shims.js',
+      ...
+    ]
+  }
+  Produces an object that looks like:
+  {
+    headerFiles: [
+     'vendor/ember-cli/vendor-prefix.js',
+     'vendor/loader/loader.js',
```

Comment:
It's a minor thing but seems like loader has to be placed right after the app-shims